src/components/container.js

```diff
@@ -316,10 +316,15 @@ function createFunctionContainer({ displayName, override } = {}) {
       });
     }
 
+    // We support renderding "bootstrap" containers without children with override API
+    // so in this case we call getCS to initialize the store globally asap
+    if (override && !containedStores.size && (scope || isGlobal)) {
```

Comment:
Should we really check if there is no `containedStores`? I think we can always bootstrap the store.

Reply:
I was more looking at the re-render cycle, where we would call the function (and then registry.has) on every re-render. Likely a micro-optimisation but given it is a simple check 🤷